flight/PiOS/Common/pios_mpu6000.c

```diff
@@ -379,21 +387,28 @@ static int32_t PIOS_MPU6000_ClaimBus()
 	if (PIOS_SPI_ClaimBus(pios_mpu6000_dev->spi_id) != 0)
 		return -2;
 
+	if (lowspeed)
+			PIOS_SPI_SetClockSpeed(pios_mpu6000_dev->spi_id, MPU6000_SPI_LOW_SPEED);
+
 	PIOS_SPI_RC_PinSet(pios_mpu6000_dev->spi_id, pios_mpu6000_dev->slave_num, 0);
 	return 0;
 }
 
 /**
  * @brief Release the SPI bus for the accel communications and end the transaction
+ * \param[in] must be true when bus was claimed in lowspeed mode
  * @return 0 if successful
  */
-static int32_t PIOS_MPU6000_ReleaseBus()
+static int32_t PIOS_MPU6000_ReleaseBus(bool lowspeed)
```

Comment:
Alternatively we could remember the speed in a global and restore it.  But I'm fine with this way too.